slack_bot/app.py

```diff
@@ -67,11 +67,33 @@ def is_today_holiday() -> bool:
 
 
 def get_token() -> str:
-    return os.environ["SLACK_TOKEN"]
+    return os.environ['SLACK_TOKEN']
 
 
-def get_all_users() -> list:
-    return json.loads(os.environ.get("USERS"))
+def get_users(client: WebClient) -> list:
+    users = json.loads(os.environ.get('USERS'))
+    return get_filtered_users(users, client)
+
+
+def get_filtered_users(users: list[str], client: WebClient) -> list[str]:
```

Comment:
```suggestion
def filter_users(users: list[str], client: WebClient) -> list[str]:
```
Das ist mal wieder Geschmackssache, aber ich benutze den `get_` Prefix gerne, wenn tatsächlich Daten irgendwo her beschafft werden, aber nicht für Transformationen oder Filterungen. In diesem Fall finde ich es sprechender, die Operation zu beschreiben (in diesem Fall dann "filter_users").